Sources/ArcGISToolkit/Components/Offline/OfflineMapViewModel.swift

```diff
@@ -41,6 +41,14 @@ class OfflineMapViewModel: ObservableObject {
         onlineMap.loadStatus == .loaded && onlineMap.offlineSettings == nil
     }
     
+    /// A Boolean value indicating whether there are downloaded map areas for the web map.
+    var hasDownloadedMapAreas: Bool? {
```

Comment:
Optional Bools are generally not recommended. The value can just be false if it can't be determined.